web/sdk/react/views/tokens/tokens-page.tsx

```
@@ -0,0 +1,239 @@
+    <Callout
+      type="accent"
+      icon={<InfoCircledIcon className={tokenStyles.tokenInfoText} />}
+      className={tokenStyles.tokenInfoBox}
+    >
+      You can now add tokens anytime to reduce next month&apos;s invoice. But this
+      won&apos;t settle any existing or overdue invoices.
+    </Callout>
+  ) : null;
+}
+
+export interface TokensPageProps { }
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)